packages/ui/atomic/create-atomic-result-component/index.js

```diff
@@ -12,6 +12,31 @@ import {cwd} from 'node:process';
 import {fileURLToPath} from 'node:url';
 
 /***************** TODO: CDX-1428: Move to @coveo/create-atomic-commons package ******************/
+class SerializableAggregateError extends AggregateError {
+  constructor(errors, message, options) {
+    super(errors, message, options);
+  }
+
+  toJSON() {
+    return {
+      name: this.name,
+      message: this.message,
+      stack: this.stack,
+      options: this.options,
+      errors: Array.from(this.errors),
+    };
+  }
+}
+
+const throwOrNotifyCli = (error) => {
```

Comment:
How about renaming this to handleErrors e.g. to make the high-level purpose clearer ?

Reply:
- [ ] rename